### ISSUE TITLE
No seeds or peers 

### ISSUE THREAD

**Issue by jcooley70** [hmaverickadams/breach-parse#8]
There are currently 0 Seeds and 0 Peers for downloading the magnet link for breach parse.  I am very curious how I can get this data, or compile my own data for the script to work with.

**Comment by acheong08** [comment]
there are a heck ton of seeds

**Comment by acheong08** [comment]
I got the file downloaded in minutes

**Comment by jcooley70** [comment]
I figured it out.   The college was blocking torrents.  I set up a vm on
azure, downloaded it there and then used vpn to get it to my kali system.
Thanks for responding.

**Comment by Vincent-vst** [comment]
I personally can't download it as well.   
I don't know if it's from my end or if it's because there is no seeder.

**Comment by jcooley70** [comment]
Turns out for me it was a firewall issue here at the university.  I was
able to utilize a vm outside this network and downloaded it just fine.

**Comment by Vincent-vst** [comment]
yeah, i think it's not a firewall issue for me as i've already downloaded plenty of torrent on the network i'm currently in
